tests/kafkatest/services/zookeeper.py

```diff
@@ -83,3 +88,11 @@ def clean_node(self, node):
 
     def connect_setting(self):
         return ','.join([node.account.hostname + ':2181' for node in self.nodes])
+
+    def set_cmd_opts(self, options):
```

Comment:
setters in python aren't as necessary as in java - it's fine to have external users do `zk.kafka_opts = my_options`

It's easy in python to refactor a field into a "property" which can encapsulate more complicated set or get logic without changing the external usage (still `zk.kafka_opts = blah`)